app/controllers/album.js

```diff
@@ -0,0 +1,36 @@
+const axios = require('axios');
+const logger = require('../../app/logger');
```

Comment:
In Wolox, we defined good practices that for order to requires : first, external dependencies, second, internal dependencies, and finally configs and constants. Use white line space for separte this.